### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by ofrobots**
* Update branding to Stackdriver Debugger.
* It is no longer a requirement that source be uploaded to the cloud
  repo.
* Provide documentation on alternative ways to make the source code
  accessible.

@matthewloring, @cristiancavalli PTAL.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +71,9 @@ Once your application is running (deployed, or elsewhere), you should be able to
 
 ![Debug UI](doc/images/debug-ui.png?raw=true)
 
-You can browse the code, and set a snapshot by clicking in the gutter (line number area). Once you set a snapshot, the debug agent will insert a momentary breakpoint at the code location in the running instances of the application.
+If your source is hosted in a [cloud source repository](https://cloud.google.com/tools/cloud-repositories/docs/), Stackdriver Debugger will display the source code of your application automatically. Alternatively, you can also point the debugger to local files, a GitHub or BitBucket repository, or through a Source Capture. More details are [available here](https://cloud.google.com/debugger/docs/source-options). Or you can simply provide a filename and line number to set a snapshot.
```
Comment (matthewloring) [inline README.md]: Parallel structure on the list here + second sentence isn't complete.

Comment (matthewloring) [inline README.md]: For branding, I think Bitbucket is more correct.

**Comment by matthewloring** [comment]
This will need rebasing to pick up the latest travis changes.

File: README.md (hunk 1 above)
Comment (ofrobots) [inline README.md]: I don't see an incomplete sentence :/

**Comment by ofrobots** [comment]
Ready for review.

**Comment by cristiancavalli** [comment]
LGTM 👍

**Pull request merged**